src/clientSideScene/CameraControls.ts

```diff
@@ -64,6 +64,27 @@ export type ReactCameraProperties =
 
 const lastCmdDelay = 50
 
+class CameraRateLimiter {
+  lastSend?: Date = undefined
+  rateLimitMs: number = 16 //60 FPS
+
+  send = (f: () => void) => {
+    let now = new Date()
+
+    if (
+      this.lastSend === undefined ||
+      now.getTime() - this.lastSend.getTime() > this.rateLimitMs
+    ) {
+      f()
+      this.lastSend = now
+    }
+  }
+
+  reset = () => {
+    this.lastSend = undefined
+  }
```

Comment:
Everything's great, it just seems reset isnt called anywhere (I'm surprised tsc or lint passed this)

Reply:
yea we may have wanted that later.  prob not now though.  feel free to remove it in a followup